### PR TITLE
Add support for foundation credential

### PR DESCRIPTION
**Pull request by jleach**
This PR add support for the BCID foundation credential.

### CODE REVIEW

**Comment by jleach** [comment]
Holding this PR in "Draft" until the accompanying changes are merged in to main Hyperledger repo first.

File: app/src/App.tsx
Comment (amanji) [inline app/src/App.tsx:1]: Thank you! This was confusing.

**Review by amanji — APPROVED** [review]

**Pull request merged**
